src/core/chemstate_mod.F90

```diff
@@ -40,13 +40,15 @@ module ChemState_Mod
    !! \param nSpeciesGas: The number of gas species.
    !! \param nSpeciesAero: The number of aerosol species.
    !! \param nSpeciesDust: The number of dust species.
-   !! \param nSpeicesSeaSalt: The number of sea salt species.
+   !! \param nSpeciesSeaSalt: The number of sea salt species.
    !! \param SpeciesIndex: An array containing the total species index.
    !! \param AeroIndex: An array containing the aerosol species index.
    !! \param GasIndex: An array containing the gas species index.
    !! \param DustIndex: An array containing the dust species index.
+   !! \param DryDepIndex: An array containing the dry deposition species index.
    !! \param SeaSaltIndex: An array containing the sea salt species index.
-   !! \param chemSpecies: A 2-D array containing the concentration of each species.
+   !! \param DryDepIndex: An array containing the dry deposition species index.
+   !! \param SpeciesNames: A character array containing the names of the species.
    !!
```

Comment:
Looks like DryDepIndex is listed twice here. For clarity, should we remove one of them?